bin/reset.imx-usb-recovery

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
This file is identical to reset.tegra-usb-recovery. Rather than duplicating it, I would suggest renaming reset.tegra-usb-recovery to reset.recovery_download, and amending all existing config files. That way, you can use it for i.MX too.